example/pubspec.lock

```diff
@@ -7,42 +7,42 @@ packages:
       name: archive
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.0.11"
+    version: "2.0.13"
   args:
```

Comment:
Upgrading the dependencies isn't the point of your PR, and It would be better if your PR contained only the changes it means to bring / fixed only the issues it means to. Can you leave out the changes to `pubspec.lock`?